src/lib.rs

```diff
@@ -135,21 +136,25 @@ impl Game {
         context.stroke();
 
         // Draw the conference table
+        context.set_stroke_style(&JsValue::from_str("#000000"));
+        context.set_fill_style(&JsValue::from_str("#358"));
         self.circle(275.0, 275.0, 75.0)?;
 
         let show_dead_people = match self.local_player() {
             None => true,
-            Some(p) => p.dead,
+            Some(p) => p.dead || p.impostor,
```

Comment:
IIUC even though only imposters can "see" dead people, every client will have that state locally.

Reply:
Yes, that's true. In the longer term we can filter data on the server